src/FuseProcess.ts

```diff
@@ -31,8 +31,9 @@ export class FuseProcess {
 				mdl;
 		}
 		return new Promise((resolve, reject) => {
-			let cache = (<any>require).cache, cached = cache[this.filePath], closePromise, exps;
-			if (cached) {
+			let cache = (<any>require).cache, closePromise, exps
+			const cached = cache && cache[this.filePath];
+			if (cached != null) {
```

Comment:
Null is quite a tricky thing, are u sure about it?

Reply:
Well I was just checking for null or undefined hence the `!=` and not `!==`. Unless you are referring to something else.

Reply:
What are the possible values of the `cache` variable?

Reply:
I have no idea cache is defined as any.

Reply:
@nchanged can you merge or reject and close this pr?

Reply:
Please don’t rush me on this one, I need to test it myself I just haven’t had time yet

Reply:
```
var cache = undefined;
cache == null // true o_O
```

what's wrong with just `if (!cache) {}` it shouldn't yield falsy values